interface/patient_file/report/patient_report.php

```diff
@@ -198,8 +311,8 @@ function show_date_fun(){
    <input type='checkbox' name='include_employer' id='include_employer' value="employer"><?php xl('Employer','e'); ?><br>
    -->
    <input type='checkbox' name='include_insurance' id='include_insurance' value="insurance"><?php xl('Insurance','e'); ?><br>
-   <input type='checkbox' name='include_billing' id='include_billing' value="billing"
-    <?php if (!$GLOBALS['simplified_demographics']) echo 'checked'; ?>><?php xl('Billing','e'); ?><br>
+   <label id="opt_fee"><input type='checkbox' name='include_billing' id='include_billing' value="billing"
+    <?php if (!$GLOBALS['simplified_demographics']) echo 'checked'; ?>><?php xl('Billing','e'); ?><br></label>
```

Comment:
In this case, since mods don't involve the strings themselves, I usually don't push for changing the xl stuff. That would be better to do in a separate project after this is done if inclined.
(so new code has to follow the security model, but I don't force conversion of old code)